modules/RTC/RTCUtils.js

```diff
@@ -24,6 +24,10 @@ var devices = {
     video: true
 };
 
+var audioOuputDeviceId = ''; // default device
+
+var featureDetectionVideoEl = document.createElement('video');
```

Comment:
Yes, makes sense. I've did this before I discovered the problems with video elements.